meson.build

```diff
@@ -358,6 +358,8 @@ possible_cc_flags = [
         '-fstack-protector',
         '-fstack-protector-strong',
         '--param=ssp-buffer-size=4',
+        '-Wvla-larger-than=128',
+        '-Walloca-larger-than=128',
```

Comment:
I think 1024 would be more reasonable or maybe even 2k would be more reasonable for VLA. We sometimes use alloca for PATH_MAX. With 128 there'll be many more warnings, and anything below a few k should be OK. (128 for alloca seems OK.).

Although, I don't get any warnings at all with gcc-9.0.1-0.10.fc30.x86_64. As @evverx said, this option doesn't seem to be working.